Framework/Geometry/src/Instrument/RectangularDetector.cpp

```diff
@@ -9,8 +9,25 @@
 #include <algorithm>
 #include <ostream>
 #include <stdexcept>
+#include <boost/regex.hpp>
 #include "MantidGeometry/Instrument/RectangularDetectorPixel.h"
 
+namespace {
+/**
+* Return the number of pixels to make a texture in, given the
+* desired pixel size. A texture has to have 2^n pixels per side.
+* @param desired :: the requested pixel size
+* @return number of pixels for texture
+*/
+int getOneTextureSize(int desired) {
```

Comment:
I am not sure why this was done in the RectangularDetector but I can change this to size_t if you prefer.